workloads/network-perf/run.sh

```diff
@@ -1,20 +1,53 @@
 #!/usr/bin/env bash
 
 source ./common.sh
-export WORKLOAD=${WORKLOAD}
 
-if [[ $WORKLOAD == "hostnet" ]]; then
+CR=ripsaw-uperf-crd.yaml
+export PAIRS=${PAIRS:-1}
+
+case ${WORKLOAD} in
+  pod2pod)
+  ;;
+  pod2svc)
+    export SERVICEIP=true
+  ;;
+  hostnet)
     export HOSTNETWORK=true
-elif [[ $WORKLOAD == "service" ]]; then
-    export SERVICEIP=true 
+  ;;
+  smoke)
+    CR=smoke-crd.yaml
+    export SAMPLES=1
```

Comment:
We should reduce the number of pairs for the smoke test as well. No need to do all 3 IMHO

Reply:
ack. figured we could save a few seconds but s'all good.

Reply:
Runtime has been 00:05:52, not that bad :)

Reply:
ack. WFM